components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2852,19 +2854,51 @@ impl ObserveID {
 
 struct ObserveCmd {
     // TODO: Add flags to disable observing.
-    id: ObserveID,
+    cdc_id: ObserveID,
```

Comment:
Once we extract the  `resolve ts` completely, is the obeserver id of the cdc still needed? Can we keep only one ?

Reply:
I think it is unneeded, the `cdc` worker and `resolved-ts` worker  will share the same observer /cc @5kbpers